apps/web/src/views/TradingReward/config/pairs/56.tsx

```diff
@@ -0,0 +1,13 @@
+import { defineFarmV3Configs } from '@pancakeswap/farms/src/defineFarmV3Configs'
```

Comment:
Better import from `@pancakeswap/farms` directly?

Reply:
you can make it exportable then